web/tests/functional/run_tag/test_run_tag.py

```diff
@@ -172,13 +172,16 @@ def test_file_change(self):
         # Check the second file version.
         self._create_source_file(1, 'test_run_tag_update')
 
+        test_run_tags = self.__get_run_tag_counts(run_id, 100, 0)
+        run_tags_v0 = [t for t in test_run_tags if t.name == self.tags[0]]
```

Comment:
are there any other run tags mixed into the test data, that is why this run tag filtering needed?